compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaAnnotationSymbol.java

```diff
@@ -74,8 +74,8 @@ private BallerinaAnnotationSymbol(String name, PackageID moduleID, Set<Qualifier
      * @return {@link Optional} type descriptor of the annotation
      */
     @Override
-    public Optional<BallerinaTypeDescriptor> typeDescriptor() {
-        return Optional.ofNullable(typeDescriptor);
+    public BallerinaTypeDescriptor typeDescriptor() {
```

Comment:
For an annotation, type descriptor is optional right?

Reply:
Ah yes. Will correct it